_includes/mobile/adunit-config-android.md

```diff
@@ -33,3 +33,13 @@ Using the following method, you can set the impression-level [GPID](https://docs
 ``` kotlin
 adUnit?.gpid = "/36117602/hnp-sfgate.com/Homepage/AP300"
 ```
+
+### Ad Position
+
+The `adPosition` property allows developers to specify the position of the ad within the publisher's content. This property maps to the `pos` field in the OpenRTB specification under the `imp[].banner` or `imp[].video` objects, depending on the ad format. The possible values for this field could be found in the [respective specification](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/main/AdCOM%20v1.0%20FINAL.md#list--placement-positions-).
+
+You can set `adPosition` by using the following method: 
+
+```kotlin
```

Comment:
Not sure if this is the right time to start this, but there is a custom component "code/mobile-sdk.html":

https://docs.prebid.org/docs-examples/tab-example.html

This would allow you to write the documentation once and put the code into this tabbing component. From my perspective it would hurt that one would see both code examples.

@bretg what are your thoughts on this?